### PR TITLE
POC for working test Tmedia 580 test working GitHub changeset (testing working github action)

### PR DESCRIPTION
**Pull request by JackHowa**
- Test that github action into canary works as expected

### CODE REVIEW

File: blocks/card-list-block/features/card-list/default.jsx [hunk 1]
```diff
@@ -179,7 +181,7 @@ const CardListItems = (props) => {
               </article>
               {contentItems.slice(1).map((element) => {
                 const {
-                  headlines: { basic: headlineText } = {},
+                  headlines: { basic: headlineText },
```
Comment (JackHowa) [inline blocks/card-list-block/features/card-list/default.jsx:184]: removed some of these fallbacks just bc they weren't tested and wanted to get a positive coverage quick

Comment (matthewroach) [reply blocks/card-list-block/features/card-list/default.jsx:184]: Removing the `= {}` will mean that the headlines key is always available with basic key

Comment (JackHowa) [reply blocks/card-list-block/features/card-list/default.jsx:184]: this is to just test a positive change in the github action

Comment (JackHowa) [reply blocks/card-list-block/features/card-list/default.jsx:184]: see note in https://github.com/WPMedia/arc-themes-blocks/pull/1210 See passing github action https://github.com/WPMedia/arc-themes-blocks/runs/4731349155?check_suite_focus=true

**Pull request closed without merging**